.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-.gitignore
\ No newline at end of file
+.gitignore
```

Comment:
This is a leftover from the git discovery phase, and doesn't make sense indeed ;).